Tests/StructuredQueriesTests/AggregateFunctionsTests.swift

```diff
@@ -247,13 +247,13 @@ extension SnapshotTests {
         """
       }
 
-      assertQuery(Tag.select { $0.name.groupConcat() }.order(by: \.name)) {
+      assertQuery(Tag.select { $0.title.groupConcat() }.order(by: \.title)) {
```

Comment:
Might be annoying, but what do you think of extracting this rename to its own PR?

Reply:
yeah agreed, can do!